precompiles/outposts/stride/stride.go

```diff
@@ -30,6 +30,7 @@ var f embed.FS
 
 type Precompile struct {
 	cmn.Precompile
+	WevmosAddress  common.Address
```

Comment:
can this be a private field instead of public?
With the current logic, will work the same as private. Unless we're planning future use cases that will require it to be public